common/static/common/js/discussion/utils.js

```diff
@@ -539,6 +542,37 @@
             };
         };
 
+        DiscussionUtil.handleKeypressInToolbar = function(event) {
+                var $currentButton, $nextButton, $toolbar, $allButtons,
+                    keyPressed, nextIndex, currentButtonIndex;
+
+                var leftKey = this.LEFT_KEYCODE;
+                var rightKey = this.RIGHT_KEYCODE;
+
+                $currentButton = $(event.target);
+                keyPressed = event.which || event.keyCode;
+                $toolbar = $(event.target).parent();
+                $allButtons = $toolbar.children('.wmd-button');
```

Comment:
What if the variable `$allButtons` does not contain any array items? What if you got no buttons?
Not very likely, but we should check for this condition to avoid surprises

Reply:
I can add the check just to make sure